src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/EmailAddressAdder.tsx

```diff
@@ -85,17 +90,17 @@ const EmailAddressAddForm = () => {
   const l10n = useL10n();
   const recordTelemetry = useTelemetry();
   const formRef = useRef<HTMLFormElement>(null);
-  const [formState, formAction] = useFormState(onAddEmail, {});
+  const [onAddEmailState, onAddEmailAction] = useActionState(onAddEmail, {});
```

Comment:
`useFormState` was renamed to `useActionState`: https://github.com/facebook/react/pull/28491